packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -377,7 +391,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
         this.socket.emit("get_ops", this.clientId, {
             nonce,
             from,
-            to,
+            to: to - 1,
```

Comment:
Should this adjustment also be made in the `this.getOpsMap.set` above?

Reply:
No, all our telemetry preserves same semantics as code - left is inclusive, right is exclusive. Values in this.getOpsMap are used only for telemetry down the road.